packages/db-base/src/lib/inMemFileDB.ts

```diff
@@ -10,6 +10,7 @@
 import fs from 'fs-extra';
 import path from 'path';
 import { tools } from '@iobroker/js-controller-common';
+import type { InternalLogger } from '@iobroker/js-controller-common/build/lib/common/tools';
```

Comment:
A subpath export would be really nice here. This works, but I prefer not to rely on internal project structures when importing from other packages.
An alternative would be to move all type definitions to a separate file and export that completely from `index.ts`